ignite3/src/main/java/site/ycsb/db/ignite3/IgniteClient.java

```diff
@@ -177,9 +200,13 @@ public Status update(String table, String key, Map<String, ByteIterator> values)
   @Override
   public Status delete(String table, String key) {
     try {
-      kvView.remove(null, Tuple.create(1).set(PRIMARY_COLUMN_NAME, key));
+      wrapWithTx(() -> kvView.remove(null, Tuple.create(1).set(PRIMARY_COLUMN_NAME, key)));
```

Comment:
Why don't we use `tx` here? Was it made intentionally or not?
```suggestion
      wrapWithTx(() -> kvView.remove(tx, Tuple.create(1).set(PRIMARY_COLUMN_NAME, key)));
```

Reply:
👍 good catch, thanks